docs/docs/frontend-protocol.md

```diff
@@ -73,9 +73,14 @@ note for `new_view`. Errors are not currently reported.
 
 `set_theme {"theme_name": "InspiredGitHub"}`
 
-Requests that core change the theme. If the change succeeds the client
```

Comment:
This is twice now someone has made this change, and twice I've complained that, actually, this is totally acceptable English, as in "we ask that they remove their shoes." Although it seems like this is confusing people, so maybe we could just go with "Asks core to change the theme"?

Reply:
No problem, will change :)